### ISSUE TITLE
litematic support

### ISSUE THREAD

**Issue by dlaverick** [EngineHub/WorldEdit#1287]
**Is your feature request related to a problem? Please describe.**
Since the change from 1.12 => 1.13+ using schemtaica to create new schematics is a no go.
Litematica is now the 'way forward', but there is no support for litematica

**Describe the solution you'd like**
Be able to paste and export to .litematic file types, just like we can with .schematic files

**Describe alternatives you've considered**
NA

**Additional context**
I know a couple of people are looking as I have, I saw on curseforge that request should be made here.

**Comment by wizjany** [comment]
I don't think we have any interest in picking this up. Schematica/litematica make some internal changes to the normal mcedit format, and we don't necessarily want to have to deal with their "proprietary" changes. WorldEdit's API allows third party plugins/mods to register additional schematic formats already, so someone else could add it via a third-party plugin/mod.

**Comment by wbradw** [comment]
Don’t see the need for this when litematica and worldedit work fine side by side anyway . Take a build/design over from one place to the other then use worldedit afterwards to move it around after pasting

**Comment by wizjany** [comment]
you seem to be assuming litematica runs everywhere worldedit does. it doesn't.

**Comment by wbradw** [comment]
I’m just saying the litematica mod works well together with worldedit. Especially worldedit has been brilliant for the community I’m part of with getting a new world set up with new spawn point and hubs with each new season we do 2- times a year. Not saying it is used everywhere just saying it’s a good pairing 👍🏻. 

The only issue we’ve noticed is when you are  copy and pasting in a build or section with corner stairs models and fences orientation with their connected neighbours is janky within the newest version of worldedit but i know it’s been addressed already 👍🏻 But other than that we’ve had no issues

**Comment by wizjany** [comment]
>  litematica mod works well together with worldedit
this is beside the point. if the player is on a bukkit server and wants to load in something that they made in singleplayer with the litematica mod, they can't. they need to save the SP build with worldedit and then load it on the server with worldedit.

re: janky stairs etc, it's "fixed" in dev builds atm by turning off validation by default, which you can do manually via //perf validation off

**Comment by wbradw** [comment]
Of course it wouldn’t work for any random player going into a server doing such but I’m a server op and admin of said community servers so litematica paste into our server world works then I can just use worldedit to move it around if needs be .

Anyway thanks for the info on the fix for the minor Paste Orientation issue  😁👍🏻

**Comment by personalized-advertising** [comment]
Guys what are we doing its 2026 and we still can't load litematic files in worldedit, 
This is easily the most valuable change that could be added for not a huge amount of effort.
@me4502 @maruohon Lets set our differences aside and not argue about which format is better, universal compatibility makes life better for builders all around the world.

**Comment by me4502** [comment]
The blocker here is a lack of a written specification for the Litematic format from their end. There are no differences to settle, we just need that before it can be implemented. I asked about it while ago in the Litematic discord and was told there's motivation to write one, but no one has gotten around to it from their end.

Once one is available I have every intention of supporting the format
